airbyte-integrations/connectors/source-bigquery/integration_tests/basic_dataset.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE
```

Comment:
The SQL files are used for manual provisioning right? Could you document / commit the scripts to provision BQ?

Reply:
Here I just stored it. As I proposed before, please give me some common place where I can document all of it for all sources.

Reply:
The script / documentation for BigQuery could go under `integration_tests` if it's bigquery specific.